client/src/components/settings/system-management/nat-getaway-configuration/add-route-modal/add-route-modal.js

```diff
@@ -43,7 +43,7 @@ export default class AddRouteForm extends React.Component {
   };
 
   state = {
-    ports: {'port': undefined},
+    ports: {'port': {value: undefined, protocol: 'tcp'}},
```

Comment:
Please define `const protocols = {TCP: 'tcp', UDP: 'udp'}` enumeration and use it.